src/renderer/themes/theme-vars.css

```diff
@@ -0,0 +1,136 @@
+
```

Comment:
I don't think this file should be committed, should it?

Reply:
Yes, it should. It's used in app as default values for themed variables + used by editor's css-vars auto-completion.

_P.S._ Most `vscode` plugins for autocompleting css-variables support only defining exact file to read for variables, so could be useful too.